release_notes/Service_Fabric_ReleaseNotes_82CU7.md

```diff
@@ -0,0 +1,60 @@
+# Microsoft Azure Service Fabric 8.2 Cumulative Update 7.0 Release Notes
+
+
+* [Service Fabric Packages and Versions](#service-fabric-packages-and-versions)
+* [Retirement and Deprecation Path Callouts](#retirement-and-deprecation-path-callouts)
+* [Repositories and Download Links](#repositories-and-download-links)
+
+## Service Fabric Packages and Versions
+This release is for Windows only.
```

Comment:
What changes ship with this release?

Reply:
couldn't get changes with release notes.

Reply:
dont have changes with release notes